### PR TITLE
Refactors timetables 

### PR DESCRIPTION
**Pull request by lightsprint09**

### CODE REVIEW

File: Timetable/TimetableLoader.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+class TimetableLoader {
+  constructor(APIToken) {
+    this.APIToken = APIToken;
+  }
+
+  timetableForEvaId(evaId) {
+    const now = moment();
+    const nowString = now.format('YYMMDD/HH');
+    const url = `${baseURL}/timetables/v1/plan/${evaId}/${nowString}`;
```
Comment (dennispost) [inline Timetable/TimetableLoader.js]: add /timetables/v1 to baseURL

File: schema.js [hunk 2]
```diff
@@ -218,17 +218,17 @@ const schema = buildSchema(`
     text: String!
   }
 
-  type ArrivalDepatureBoard {
-	  nextArrivals: [TrainInStation]
-	  nextDepatures: [TrainInStation]
+  type Timetable {
+	  nextArrivals: [TrainInStation!]!
```
Comment (dennispost) [inline schema.js:242]: Can you say for sure there is always a departure / arrival train?

File: schema.js [hunk 3]
```diff
@@ -218,17 +218,17 @@ const schema = buildSchema(`
...
-	  type: String
-	  trainNumber: String
-	  platform: String
-	  time: String
-	  stops: [String]
+	  type: String!
+	  trainNumber: String!
+	  platform: String!
+	  time: String!
+	  stops: [String!]!
```
Comment (dennispost) [inline schema.js:251]: Are there only trains with at least one stop in stops?

File: schema.js (hunk 2 above)
Comment (lightsprint09) [reply schema.js:242]: yes. If not it will be an empty array

**Pull request merged**